static/js/components/CommentVoteForm.js

```diff
@@ -20,32 +22,55 @@ export default function CommentVoteForm(props: Props) {
   const [upvoting, setUpvoting] = useState(false)
   const [downvoting, setDownvoting] = useState(false)
 
+  const showErrorBanner = useCallback(
```

Comment:
Should we display an error banner when upvoting posts fails as well?

Reply:
probably - but I think I'm going to tackle that in a separate PR. I'm sort of just going through all this older code in open when I have time and trying to fix it up :)